packages/module/patternfly-docs/content/extensions/component-groups/examples/ErrorState/ErrorStateFooterExample.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import { Button } from '@patternfly/react-core';
+import ErrorState from "@patternfly/react-component-groups/dist/dynamic/ErrorState";
+
+// eslint-disable-next-line no-console
+export const BasicExample: React.FunctionComponent = () => <ErrorState errorTitle='Sample error title' errorDescription='Sample error description' customFooter={<Button variant="secondary" onClick={() => console.log("Custom button clicked")}>
```

Comment:
Do we want to block the `onClick` and console log the `"button clicked?"`

Reply:
It's an example for the docs

Reply:
Oh, yeah! Feel free to merge!